bolt/lib/Passes/ProfileQualityStats.cpp

```diff
@@ -157,8 +157,10 @@ void printCFGContinuityStats(raw_ostream &OS,
     FractionECUnreachables.push_back(FractionECUnreachable);
   }
 
-  if (FractionECUnreachables.empty())
+  if (FractionECUnreachables.empty()) {
+    OS << "function CFG discontinuity 0.00%; ";
```

Comment:
The issue is due to single-BB functions, right? Can we just assume the discontinuity is 0 for them, instead of skipping in L94-95? And apply a similar logic to other metrics.